Makefile

```diff
@@ -201,19 +201,19 @@ avx.c          := $(sort $(wildcard backends/avx/*.c))
 xsmm.c         := $(sort $(wildcard backends/xsmm/*.c))
 cuda.c         := $(sort $(wildcard backends/cuda/*.c))
 cuda.cpp       := $(sort $(wildcard backends/cuda/*.cpp))
-cuda.cu        := $(sort $(wildcard backends/cuda/*.cu))
+cuda.cu        := $(sort $(wildcard backends/cuda/kernels/*.cu))
```

Comment:
@nbeams and I talked about this idea in Slack. It looks like the new OCCA backend will be similarly organized.